src/matrix.rs

```diff
@@ -101,14 +101,29 @@ impl<S> Matrix2<S> {
 impl<S: BaseFloat> Matrix2<S> {
     /// Create a transformation matrix that will cause `unit_x()` to point at
     /// `dir`. `unit_y()` will be perpendicular to `dir`, and the closest to `up`.
+    #[deprecated = "Use Matrix2::look_to"]
     pub fn look_at(dir: Vector2<S>, up: Vector2<S>) -> Matrix2<S> {
-        Matrix2::look_at_stable(dir, up.x * dir.y >= up.y * dir.x)
+        Matrix2::look_to(dir, up)
     }
 
     /// Crate a transformation that will cause `unit_x()` to point at
-    /// `dir`. This is similar to `look_at`, but does not take an `up` vector.
+    /// `dir`. This is similar to `look_to`, but does not take an `up` vector.
     /// This will not cause `unit_y()` to flip when `dir` crosses over the `up` vector.
+    #[deprecated = "Use Matrix2::look_to_stable"]
     pub fn look_at_stable(dir: Vector2<S>, flip: bool) -> Matrix2<S> {
+        Matrix2::look_to_stable(dir, flip)
+    }
+
+    /// Create a transformation matrix that will cause `unit_x()` to point at
+    /// `dir`. `unit_y()` will be perpendicular to `dir`, and the closest to `up`.
+    pub fn look_to(dir: Vector2<S>, up: Vector2<S>) -> Matrix2<S> {
```

Comment:
This should be `look_to_{lh|rh}`. I didn't update these because I didn't know what the current implementation was.

Reply:
Is there a handedness distinction for 2D rotations?

Reply:
I think that there is because handedness defines the direction of rotation. However, I had the same (unresolved) question which is why I decided not to modify it in the other PR. My reasoning was that it's probably better to leave it alone until I (or someone else) has time to figure out the correct path forward.

Reply:
I'm confused. I thought handedness determines between coordinate systems. The direction of rotation should be indistinguishable here. I.e. the rotation is the same regardless of which way you turn. For 2D rotations, setting an upside-down up vector should cause the coordinate system to flip, which, I think does the equivalent of what the 3D handedness does. I.e. it flips the sign of the output matrix determinant. Maybe I'm missing something here? It seems like we can either do handedness here without the `up` vector or leave it as is.